packages/miniflare/test/index.spec.ts

```diff
@@ -2995,6 +2995,46 @@ test("Miniflare: CF-Connecting-IP is preserved when present", async (t) => {
 	t.deepEqual(await ip.text(), "128.0.0.1");
 });
 
+// regression test for https://github.com/cloudflare/workers-sdk/issues/7924
+test("Miniflare: strips CF-Connecting-IP", async (t) => {
+	const server = new Miniflare({
+		script:
+			"export default { fetch(request) { return new Response(request.headers.get(`CF-Connecting-IP`)) } }",
+		modules: true,
+	});
+	const url = await server.ready;
```

Comment:
NIT: it would be helpful to give this a more context specific name:

```suggestion
	const serverUrl = await server.ready;
```